kie-internal/pom.xml

```diff
@@ -87,6 +87,7 @@
               org.kie.api.logger,
               org.kie.api.marshalling,
               org.kie.api.persistence.jpa,
+              org.kie.api.pmml,
```

Comment:
@mariofusco 
Out of curiosity, why are those modifications needed in the Import-Package ?